docs/auth-engines.md

```diff
@@ -92,3 +93,104 @@ This CR is roughly equivalent to this Vault CLI command:
 ```shell
 vault write [namespace/]auth/kubernetes/role/database-engine-admin bound_service_account_names=vaultsa bound_service_account_namespaces=<dynamically generated> policies=database-engine-admin
 ```
+
+## LDAPAuthEngineConfig
+
+The `LDAPAuthEngineConfig` CRD allows a user to configure an authentication engine mount of [type LDAP](https://www.vaultproject.io/docs/auth/ldap).
+
+```yaml
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: LDAPAuthEngineConfig
+metadata:
+  name: authenginemount-sample
+spec:
+  UPNDomain:
+  anonymousGroupSearch:
+  authentication: 
+    path: kubernetes
+    role: policy-admin
+    serviceAccount:
+      name: admin-sa
+  bindDN: cn=vault,ou=Users,dc=example,dc=com
+  bindPass: xxxxxxxxxxxxx
+  caseSensitiveNames: false
+  groupDN:
+  groupFilter:
+  insecureTls: true
+  path: ldap
+  tokenReviewerServiceAccount:
```

Comment:
I don't understand what this is used for, can you explain? it looks like it's not a field of the LDAP secret engine API.

Reply:
I removed this. I thought it was used in the Authentication part, but I was mistaken. Documentation has been updated accordingly.